apps/src/netsim/NetSimLocalClientNode.js

```diff
@@ -365,6 +365,7 @@ NetSimLocalClientNode.prototype.disconnectRemote = function (onComplete) {
     // re-disconnect.
     if (err) {
       logger.info("Error while disconnecting: " + err.message);
+      onComplete(err);
     }
```

Comment:
See the comment above; we intentionally don't `return` in the error clause here, so we don't need to add an `onComplete` call because we fall through the the one below (line 372).